### PR TITLE
Made a Project class, added a barebones VisualElement interface

### PR DESCRIPTION
**Pull request by AlexMirabella**
Made a Project class, added a barebones VisualElement interface

### CODE REVIEW

File: src/main/java/org/openjfx/FXMLController.java [hunk 1]
```diff
@@ -45,9 +46,7 @@ private  List<Project> projects() {
         List<Project> ls = new ArrayList<Project>();
 
         for (int i = 0; i < 15; i++) {
-            Project project = new Project();
-            project.setName("Untitled Project");
-            project.setDate("Modified July 20");
+            Project project = new Project("Untitled Project");
```
Comment (latxh) [inline src/main/java/org/openjfx/FXMLController.java:48]: I like the addition of taking in the Project Name

File: src/main/java/org/openjfx/FXMLController.java [hunk 2]
```diff
@@ -14,6 +14,7 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.ResourceBundle;
+import java.util.Date;
```
Comment (latxh) [inline src/main/java/org/openjfx/FXMLController.java]: Remove unused import

**Comment by latxh** [comment]
Also add a description , besides that LGTM

**Review by latxh — APPROVED** [review]

**Review by SquaredPiano — APPROVED** [review]
Great updates to project class! LGTM

**Pull request merged**